.eslintrc.cjs

```diff
@@ -8,6 +8,11 @@ module.exports = {
     "@electron-toolkit/eslint-config-ts/recommended",
     "plugin:prettier/recommended",
   ],
+  settings: {
```

Comment:
I was getting build errors if this wasn't done. If it works without it, don't hesitate to remove it.